app/controllers/rents_controller.rb

```diff
@@ -10,11 +11,16 @@ def index
   end
 
   def create
-    book = Book.find(params[:rent][:book_id])
-    render json: Rent.create(user: current_user, book: book,
+    @book = Book.find(params[:rent][:book_id])
+    @user = current_user
```

Comment:
You're assigning `current_user` to `@user`, but using `current_user` later when creating a new Rent. Is this assignment necessary?

Reply:
I put it to send the data to the mailer.

Reply:
Did you try if the current_user method worked in `sendmail`?

Reply:
yes and it didn't